frontend/packages/pipelines-plugin/src/components/repository/list-page/__tests__/RepositoryList.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { ComponentProps, Table } from '@console/internal/components/factory';
+import RepositoryList from '../ReppositoryList';
+import { mockRepository } from './repository-test-data';
+
+describe('RepositoryList', () => {
+  type RepositoryListProps = React.ComponentProps<typeof RepositoryList>;
+  let wrapper: ShallowWrapper<RepositoryListProps>;
+  const props: RepositoryListProps = {
+    data: [mockRepository],
+  };
+
+  beforeEach(() => {
+    wrapper = shallow(<RepositoryList {...props} />);
+  });
+
+  it('should render Repository list Table', () => {
+    expect(wrapper.find(Table).exists()).toBe(true);
+  });
+
+  it('should render the proper header', () => {
+    const expectedRepositoryTableHeader: string[] = [
+      'Name',
+      'Event type',
+      'Last run',
+      'Task status',
+      'Last run status',
+      'Last run time',
+      'Last run duration',
+    ];
+    const headers = wrapper
+      .find(Table)
+      .props()
+      .Header({} as ComponentProps);
+    expectedRepositoryTableHeader.forEach((header, i) => {
+      expect(headers[i].title).toBe(header);
+    });
+  });
+});
```

Comment:
What are we testing here? Isn't this all static content? What value do you think these tests bring?